TUnit.Templates.Tests/Snapshots/InstantiationTest.TUnit.AspireStarter._.verified/TUnit.AspireStarter/TUnit.AspireStarter.TestProject/Hooks.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System.Diagnostics;
+
+namespace TUnit.AspireStarter.TestProject;
+
+public class Hooks
+{
+    [Before(TestSession)]
+    public static void InstallPlaywright()
+    {
+        if (Debugger.IsAttached)
+        {
+            Environment.SetEnvironmentVariable("PWDEBUG", "1");
+        }
+
+        Microsoft.Playwright.Program.Main(["install"]);
```

Comment:
Copy and paste from the Playwright template? We don't need playwright here do we?